src/core/model_config.proto

```diff
@@ -474,6 +474,15 @@ message ModelOptimizationPolicy
     //@@       backend.
     //@@
     bool graphs = 1;
+
```

Comment:
For this and for 'graphs' above can you put that the default is "false". That is the protobuf default but still useful to document it explicitly.